science/gwyddion/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                gwyddion
-version             2.59
+version             2.62
 revision            1
```

Comment:
revision should be reset to zero when updating the version

Reply:
please don't comment out the line, just set it to zero as I said before

Reply:
Now it's zero